### DISCUSSION TITLE
Release version 4.10.2 and move the remaining milestones to 4.10.3

### DISCUSSION THREAD

**Discussion by Faar400** [CONTENIDO/CONTENIDO#381] [General]
I think, the Develop Version is good enough for a new release 4.10.2 and the missing milestones should be moved to the next release.
It is like a minor release from 10.1 to 10.2 but in reality Contenido works already with PHP 8.1 and MySQL 8.
What about my idea?

**Comment by muratpurc** [comment]
Good idea, who gives the go-ahead for it?

**Reply by gonatan** [reply]
I guess it's yours :)

**Reply by muratpurc** [reply]
@gonatan I've moved all milestone 4.10.2 related tickets to milestone 4.10.3, there is nothing left for milestone 4.10.2.

Someone else should handle the release. I don't want to take the responsibility of it. Developing features or fixing bugs is a different thing than releasing a new version.

**Reply by Faar400** [reply]
And what is the conclusion?

**Reply by dceonline** [reply]
@Faar400 you will do the release. ;)

**Reply by Faar400** [reply]
Could I ?

**Comment by Faar400** [comment]
As soon as compatibility with PHP 8.2 comes, we should call this version 4.10.3
4.10.2 would have been PHP 8.0 or 8.1
What's about a 4.11 ? Are there Milestones?

**Comment by muratpurc** [comment]
@Faar400 The last released version was 4.10.1, next would be 4.10.2.

However, if we want to follow the semantic versioning principle, the next version should be 4.11.0, as we have also added some new features since version 4.10.1.

That would be "more correct", but then we have to clarify how we can solve this with the forum. It makes no sense to create a new area (CONTENIDO 4.9, CONTENIDO 4.10, CONTENIDO 4.11, etc.) for each minor version, that will be too much. Modules and plugins developed for CONTENIDO 4.9 and/or 4.10 would still be compatible with the new version of CONTENIDO. Issues are more likely to occur with PHP 8 compatibility. The more areas we have in the forum, the more confusing it becomes for the community.

Since version 4.10.2 has not been released yet, we can stick with it and use version number 4.10.2 for the adaptation to PHP 8.2 if we do not release version 4.10.2 beforehand.

If you ask me, I would strictly follow SemVer, which is the standard nowadays.

**Reply by Faar400** [reply]
That time of 4.9 i thought already, it would have been better to call it 5.0
It was a profound change in the core and what other reason would there have been to declare a 5.0?
So let's go on with 4.10.2
